geomesa-fs/geomesa-fs-tools/src/main/scala/org/locationtech/geomesa/fs/tools/ingest/FsIngestCommand.scala

```diff
@@ -80,10 +82,13 @@ object FsIngestCommand {
                                   libjarsPaths: Iterator[() => Seq[File]],
                                   numLocalThreads: Int,
                                   tempPath: Option[Path],
-                                  reducers: Option[java.lang.Integer])
-      extends ConverterIngest(sft, dsParams, converterConfig, inputs, mode, libjarsFile, libjarsPaths, numLocalThreads) {
+                                  reducers: Option[java.lang.Integer],
+                                  jobStatusTracking: Boolean,
```

Comment:
I don't think we need this flag though - it's easy enough to accomplish the same thing with shell redirects, so we don't need to complicate the code